src/main/java/com/crud/tasks/service/SimpleEmailService.java

```diff
@@ -29,6 +31,10 @@ public void send(final Mail mail) {
     private SimpleMailMessage createMailMessage(final Mail mail) {
         SimpleMailMessage mailMessage = new SimpleMailMessage();
         mailMessage.setTo(mail.getMailTo());
+        if (Optional.ofNullable(mail.getToCc()).isPresent()) {
```

Comment:
To jest antywzorze. Metoda getToCC() powinna zwracac Optionala. Tutaj opakowujesz wartosc w Optionala, by sprawdzic czy jest pusty :/ bez sensu.